tchannel/tracing.py

```diff
@@ -97,7 +97,11 @@ def span_in_context(self, span):
         :param span: an OpenTracing Span
         :return: ``StackContext``-based context manager
         """
-        return opentracing_instrumentation.span_in_stack_context(span)
+        if isinstance(opentracing.tracer.scope_manager,
+                      opentracing_instrumentation.request_context.TornadoScopeManager):  # noqa
+            return opentracing_instrumentation.span_in_stack_context(span)
+
+        return opentracing_instrumentation.span_in_context(span)
```

Comment:
there doesn't appear to be a difference between the if and else cases. Is this a leftover from before?

Reply:
span_in_stack_context vs span_in_context